src/components/PlanCard/stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import { PlanCard } from '.'
+import { plans } from './mock'
+
+export default {
+  title: 'Components/PlanCard',
+  component: PlanCard,
+  parameters: {
```

Comment:
Adicionar background dark
```suggestion
  parameters: {
    backgrounds: {
        default: 'dark',
      },
```